src/main/java/com/exactpro/th2/FixHandler.java

```diff
@@ -1356,6 +1497,12 @@ private void applyNextStrategy() {
             ruleErrorEvent(strategy.getState().getType(), e);
         }
 
+        if(!sessionActive.get()) {
+            strategy.resetStrategyAndState(new RuleConfiguration(RuleType.DEFAULT, Duration.of(10, ChronoUnit.MINUTES), Duration.of(1, ChronoUnit.SECONDS), null, false, null, null, null, null, null, null, null));
```

Comment:
Maybe we can move this into a static method? Like this `RuleConfiguration.default()`